lib/spyke/http.rb

```diff
@@ -105,7 +105,12 @@ def add_errors_to_model(errors_hash)
         errors_hash.each do |field, field_errors|
           field_errors.each do |error_attributes|
             error = NormalizedValidationError.new(error_attributes)
-            errors.add(field.to_sym, error.message, error.options)
+
+            if errors.method(:add).arity == -2
```

Comment:
Coverage has decreased slightly, this may benefit from a matrix-style build where we express the older Rails version.